README.md

```diff
@@ -50,11 +84,15 @@ cd build
 make
 cd unit_tests  # run unit tests
 ctest
```

Comment:
You can actually do:

```
cd build/unit_tests
make
ctest
```
Which I think is neater. Might also be nice to tell our users about the `-j <n>` flag

Reply:
I think leaving out -j is fine, makes it simpler and works fine without